README.md

```diff
@@ -22,140 +43,151 @@ The following is a set of guidelines that you can use today as we enhance the ex
 
 Here's how to build the container image with the Controller Agent inside, based on the official NGINX image:
 
-Clone this repository
+1. Clone this repository:
 
-```bash
-git clone https://github.com/nginxinc/docker-nginx-controller.git
-```
+    ```bash
+    git clone https://github.com/nginxinc/docker-nginx-controller.git
+    ```
 
-```bash
-cd docker-nginx-controller/<distribution>
-```
+    ```bash
+    cd docker-nginx-controller/<distribution>
+    ```
 
-Copy your NGINX Plus repository certificate and key to the folder of the Dockerfile you will be using for your Linux distribution.
+2. Copy your NGINX Plus repository certificate and key to the folder of the Dockerfile you will be using for your Linux distribution.
 
-Edit the Dockerfile with your API_KEY and CONTROLLER_URL
+3. Edit the Dockerfile with your API_KEY and CONTROLLER_URL.
 
-(**Note**: RE: CONTROLLER_URL format
+    > Note: Use the `CONTROLLER_URL` format that's supported by your version of NGINX Controller:
+    >
+    > NGINX Controller v3.10 or older: `CONTROLLER_URL=https://<fqdn>:8443/1.4/install/controller`
+    >
+    > NGINX Controller v3.11 or newer: `CONTROLLER_URL=https://<fqdn>/install/controller-agent`
 
-```bash
-# If NGINX Controller version is 3.10 or older
-CONTROLLER_URL=https://<fqdn>:8443/1.4/install/controller/
+    For example:
 
-# If NGINX Controller version is 3.11 or newer
-CONTROLLER_URL=https://<fqdn>/install/controller-agent
-```
+    ```bash
+    sudo docker build --build-arg CONTROLLER_URL=https://<fqdn>/install/controller-agent --build-arg API_KEY='abcdefxxxxxx' -t nginx-agent .
+    ```
 
-Examples are shown using the latest version)
+4. After the image is built, view the list of Docker images:
 
-```bash
-sudo docker build --build-arg CONTROLLER_URL=https://<fqdn>/install/controller-agent --build-arg API_KEY='abcdefxxxxxx' -t nginx-agent .
-```
+    ```bash
+    sudo docker images
+    ```
 
-After the image is built, check the list of Docker images:
+    The output looks similar to the following example:
 
-```bash
-sudo docker images
-```
+    ```bash
+    REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+    nginx-agent       latest              d039b39d2987        3 minutes ago       241.6 MB
+    ```
 
-```bash
-REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
-nginx-agent       latest              d039b39d2987        3 minutes ago       241.6 MB
-```
+### 2.2. Running an NGINX Controller-Enabled NGINX Docker Container
 
-### 2.2. Running an NGINX Controller-enabled NGINX Docker Container
+Take the following steps to run an NGINX Controller-enabled NGINX Docker Container:
 
-To start a container from the new image, run the following command:
+1. To start a container from the new image, run the following command:
 
-```bash
-docker run --name mynginx1 -d nginx-agent
-```
+    ```bash
+    docker run --name mynginx1 -d nginx-agent
```

Comment:
we should be consistent with prepending command lines with 'sudo'
This is to drive home the security practice of not doing su to operate as root.